lib/msf/core/exploit/local/persistence.rb

```diff
@@ -0,0 +1,26 @@
+# -*- coding: binary -*-
+
+module Msf
+  module Exploit::Local::Persistence
+    def initialize(info = {})
+      super(
+        update_info(
+          info,
+          'DefaultOptions' => {
+            'WfsDelay' => 90_000, # 25hrs
+            'AllowNoCleanup' => true # don't delete our persistence after we get a shell
+          },
+          # https://github.com/rapid7/metasploit-framework/pull/19676#discussion_r1907594308
+          'Stance' => Msf::Exploit::Stance::Passive
+          # 'Passive' => true # XXX when set, ignores wfsdelay and immediately exists after last command
```

Comment:
I think we'd want `WfsDelay` to be ignored so the module runs in the background indefinitely or until it's stopped.

Reply:
I agree with the above, just adding a little bit of extra context:
https://docs.metasploit.com/docs/development/developing-modules/guides/get-started-writing-an-exploit.html#non-required-fields

When setting `'Passive' => true` the module will exit but you should be able to see the handler running as a job in the background with`jobs -l`.

Reply:
Its true and does, but **not** when the modules are moved to the persistence folder.